src/platforms/common/configuration/options.mdx

```diff
@@ -364,6 +364,14 @@ This is the maximum number of properties or entries that will be included in any
 
 </ConfigKey>
 
+<ConfigKey name="attach-screenshot" supported={["android"]}>
+
+Take a screenshot of the application when an error happened and include as an attachment.
```

Comment:
I've adjusted the verb tense a bit to match the wording of some of other options.
```suggestion
Takes a screenshot of the application when an error happens and includes it as an attachment.
```